Secrets/KeysExample.swift

```diff
@@ -1,6 +1,74 @@
-struct Keys {
-    struct Parse {
-        static let applicationId = ""
-        static let clientKey = ""
+/**
+    map a function over an optional value
+    - If the value is `.None`, the function will not be evaluated and this will return `.None`
+    - If the value is `.Some`, the function will be applied to the unwrapped value
+    - parameter f: A transformation function from type `T` to type `U`
+    - parameter a: A value of type `Optional<T>`
+    - returns: A value of type `Optional<U>`
+*/
+public func <^> <T, U>(@noescape f: T -> U, a: T?) -> U? {
+    return a.map(f)
+}
+
+/**
+    apply an optional function to an optional value
+    - If either the value or the function are `.None`, the function will not be evaluated and this will return `.None`
```

Comment:
Length Violation: Line should be 100 characters or less: currently 118 characters